cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/local/LocalStorage.java

```diff
@@ -0,0 +1,70 @@
+package com.linkedin.openhouse.cluster.storage.local;
+
+import com.linkedin.openhouse.cluster.storage.Storage;
+import com.linkedin.openhouse.cluster.storage.StorageClient;
+import com.linkedin.openhouse.cluster.storage.StorageType;
+import com.linkedin.openhouse.cluster.storage.configs.StorageProperties;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+/**
+ * The LocalStorage class is an implementation of the Storage interface for local storage. It uses a
+ * LocalStorageClient to interact with the local file system.
+ */
+@Component
+public class LocalStorage implements Storage {
+
+  private static final StorageType.Type TYPE = StorageType.LOCAL;
+
+  @Autowired private StorageProperties storageProperties;
+
+  // Lazy initialization of the LocalStorageClient
+  @Autowired @Lazy private LocalStorageClient localStorageClient;
+
+  /**
+   * Check if the local storage is active.
```

Comment:
This will help us configure localStorage if no storages are provided in the cluster.yaml

Reply:
yes! that was exactly the purpose. Having table service bootup without a cluster.yaml with certain default settings. `:tables-test-fixtures` also boots up without cluster.yaml

Reply:
👍